queries.yaml

```diff
@@ -23,4 +23,5 @@ queries:
     max: 0
   - title: Within due-date (10 days)
     query: project_id=36&utf8=%E2%9C%93&set_filter=1&sort=updated_on%3Adesc&f%5B%5D=status_id&op%5Bstatus_id%5D=o&f%5B%5D=subproject_id&op%5Bsubproject_id%5D=*&f%5B%5D=due_date&op%5Bdue_date%5D=%3Ct-&v%5Bdue_date%5D%5B%5D=10&f%5B%5D=last_updated_by&op%5Blast_updated_by%5D=%21&v%5Blast_updated_by%5D%5B%5D=40859&f%5B%5D=&c%5B%5D=subject&c%5B%5D=project&c%5B%5D=status&c%5B%5D=assigned_to&c%5B%5D=due_date&c%5B%5D=updated_on&c%5B%5D=category&group_by=&t%5B%5D=
+    comment: This ticket is 10 days after the due-date. Please consider closing this ticket or move the due-date accordingly.
```

Comment:
This would not only affect low tickets, right? However, I suppose the new message makes generally sense.

Reply:
No, the query is not including any filter for priority

Reply:
The query is not about priority either, this is only evaluates the due-date so yes, this will (already is) target all tickets.